packages/@react-spectrum/dropzone/stories/DropZone.stories.tsx

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {action} from '@storybook/addon-actions';
+import {Button} from '@react-spectrum/button';
+import {classNames} from '@react-spectrum/utils';
+import {Content} from '@react-spectrum/view';
+import dndStyles from '@react-aria/dnd/stories/dnd.css';
+import {DropZone} from '../';
+import {FileDropItem, FocusRing, mergeProps, TextDropItem, useButton, useClipboard, useDrag} from 'react-aria';
+import {FileTrigger} from 'react-aria-components';
+import {Heading} from '@react-spectrum/text';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+import {Link} from '@react-spectrum/link';
+import {Meta} from '@storybook/react';
+import React, {useState} from 'react';
+import ShowMenu from '@spectrum-icons/workflow/ShowMenu';
+import {SpectrumDropZoneProps} from '../src/DropZone';
+import Upload from '@spectrum-icons/illustrations/Upload';
+
+type StoryArgs = SpectrumDropZoneProps;
+
+const meta: Meta<StoryArgs> = {
+  title: 'DropZone',
+  component: DropZone
+};
+
+export default meta;
+
+export const withDraggable = {
+  render: (args) => (
+    <DropZoneWithDraggable {...args} />
+  )
+};
+
+export const withLink = {
+  render: (args) => (
+    <DropZoneWithLink {...args} />
+  )
+};
+
+export const withButton = {
```

Comment:
IMO we can remove the Button examples if there aren't any Spectrum designs for it. The Spectrum use case might be limited to a Link.

Reply:
i think this is something that we can maybe bring up with design and see if they see any use cases with a button? since it's unclear, i'll just leave it for now although you're right, currently there is no spectrum designs with a button, but we can easily choose to remove them later if needed